### PR TITLE
Fix Jubilife Muffin full heal effect + better full heal tests

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Fixes Jubilife Muffin not actually working, due to a missing effect.
Also adds Jubilife Muffin to local specialty and full heal tests, as well as adding frostbite to the test which was missing.

## **Discord contact info**
kittenchilly

### CODE REVIEW

**Review by Pawkkie — APPROVED** [review]
I've read this over twice and everything looks excellent to me, I'm gonna leave it here for a bit in case another maintainer has feedback as test reviews undermine my confidence for some reason still but will merge it shortly if nothing is pointed out. Great catch! :)

**Review by AsparagusEduardo — APPROVED** [review]
lgtm 👍

**Pull request merged**
